src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -733,6 +733,16 @@ private void Get_CORINFO_SIG_INFO(MethodDesc method, CORINFO_SIG_INFO* sig, Meth
                 {
                     sig->sigInst.methInst = GetJitInstantiation(method.Instantiation);
                 }
+
+                Instantiation methodInst = method.Instantiation;
```

Comment:
Left over from previous iteration? `sig->sigInst.methInst` is initialized a few lines above.

Reply:
Looks like it, yes.

The logic just above was added in https://github.com/dotnet/runtime/pull/61079. Where-as I had initially prototyped this work well over a year ago, but had shelved it until `Unsafe` was moved down into `Corelib` and only recently picked it back up.

Reply:
Removed the duplicate logic.